# Trajectory parameters for command_node

duration: &time 25

liss_default: &liss_default
  ampX: 0.10
  ampY: 0.20
  ampZ: 0.2
  freqX: 0.2
  freqY: 0.1 # must be different from 0
  freqZ: -0.1
  phiX : 0
  phiZ : 0
  offX : 0.0
  offY : 0.0
  offZ : 0.0
  duration: *time

liss: &liss
  <<: *liss_default  # Copia tutti i valori da liss1 usando l'anchor
  ampZ: 0.0
  freqZ: 0.0

circ: &circ
  <<: *liss_default  # Copia tutti i valori da liss1 usando l'anchor
  ampZ: 0.0
  freqX: 0.05
  freqY: 0.05 
  freqZ: 0.0

minjerk:
  xf: 0.311613
  yf: 0.0
  zf: 0.588755
  duration: 10

lissajous: *liss

flag:
  type: 1 # 1: lissajous, 2: minjerk, 3: script
